src/app/listing-page/listing-page.component.html

```diff
@@ -6,7 +6,7 @@
     </div>
     <div class="listing-text">
       <div class="listing-name">
-        <a href="#">{{listing.name}}</a>
+        <a href="http://localhost:4200/listing">{{listing.name}}</a>
```

Comment:
There's got to be a simple way to do this dynamically, right?

Reply:
yup, working on full stack listing functionality on my local branch and i changed it there

Reply:
i'll update this branch soon though